python/testEditorInterface.py

```diff
@@ -43,6 +43,12 @@ def main():
         # Contents of the line from the editor so far
```

Comment:
You probably did not want to tear this completely apart but I would probably convert this to use argparse now and have named parameters like --mode, --enviroName, --inputLine, --unit etc. and make unit optional, the others required.

(if it turns out we need unitName ... which it might)

Reply:
Refactored to argument parser in 78bf51f72c4d2f1a11d3f97b4d1575e0ba69e717 .